spa/src/components/common/RedirectWithReload.js

```diff
@@ -0,0 +1,9 @@
+import { useHistory, withRouter } from 'react-router-dom';
+
+const RedirectWithReload = (props) => {
+  let history = useHistory();
+  history.push({ pathname: props.to, search: props.location.search });
+  window.location.reload();
+  return <></>;
+};
+export default withRouter(RedirectWithReload);
```

Comment:
Can add PropTypes to make it clearer what each prop is
 ```suggestion
 RedirectWithReload.propTypes = {
     to: PropTypes.string.isRequired,
     location: PropTypes.any,
     history: PropTypes.any,
 }
 
export default withRouter(RedirectWithReload);
```

Reply:
Addressed with 02f12cff0cdaa189fe4b9b93678a7b00068cf129